src/views/payment-sheet-views/card-view.js

```diff
@@ -204,11 +206,11 @@ CardView.prototype._generateHostedFieldsOptions = function () {
     }
   };
 
-  if (!hasCVVChallenge) {
+  if (shouldNotCollectCVV) {
     delete options.fields.cvv;
   }
```

Comment:
Could have some nice symmetry with the next few lines if `shouldNotCollectCVV` was `!shouldCollectCVV` instead.